api/authenticator.py

```diff
@@ -1484,15 +1483,13 @@ def authentication_provider_document(self, _db):
             self.password_label,
             self.password_label
         )
-        method_doc = dict(
+        return dict(
+            description=unicode(self.DISPLAY_NAME),
```

Comment:
We should probably change this wording to be more imperative, like "Log in using your library card" or something.